### ISSUE TITLE
如何使抓到的cookie一直有效

### ISSUE THREAD

**Issue by ysw774977** [MikeWang000000/GoLibCookie#5]
大佬您好，我是小白，在使用过程中抓到的cookie只能维持2个小时，之后再用这个cookie就无效了，有什么好的方法只需要抓一次包吗

**Comment by MikeWang000000** [comment]
#4 重复

> 如果您使用 [Requests](http://github.com/requests/requests) 库，请使用`requests.Session()`维持会话。

每隔一段时间，使用同一 session 向 api 发起请求。  
当 cookie 临界过期时，会收到来自 api 的`Set-Cookie` 头部，包含新的 cookie，原 cookie 作废。  
requests 库的 Session 会自动维护更新 cookie 值。

**Comment by EJianZQ** [comment]
> #4 重复
> 
> > 如果您使用 [Requests](https://github.com/requests/requests) 库，请使用`requests.Session()`维持会话。
> 
> 每隔一段时间，使用同一 session 向 api 发起请求。 当 cookie 临界过期时，会收到来自 api 的`Set-Cookie` 头部，包含新的 cookie，原 cookie 作废。 requests 库的 Session 会自动维护更新 cookie 值。

也就是说我如果用别的语言和普通的post请求的话，是需要手动将set-cookie头部取出来的吗？

**Comment by MikeWang000000** [comment]
> 也就是说我如果用别的语言和普通的post请求的话，是需要手动将set-cookie头部取出来的吗？

是的。如果使用其他语言自行处理会话，需要严格按照服务器发来header中Set-Cookie的指示，调整下一次请求header中的Cookie值。
